expected/15.0/type.out

```diff
@@ -499,9 +499,381 @@ SELECT * FROM "type_DOUBLE"; -- OK
+----+--------------------------------------+------+----
+  1 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | text | 36
+  2 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | text | 36
+  3 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | text | 38
+  4 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | text | 32
+  5 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | text | 39
+  6 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | text | 37
+ 13 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | blob | 16
+ 15 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | blob | 16
+ 17 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | text | 36
+ 18 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | text | 36
+ 19 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | text | 36
+ 20 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | text | 36
+ 21 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | text | 36
+ 22 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | text | 36
+ 29 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | blob | 16
+ 30 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | blob | 16
+ 31 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | blob | 16
+ 32 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | blob | 16
+ 33 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | blob | 16
+ 34 | a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11 | blob | 16
+(20 rows)
+
+--Testcase 168:
+ALTER FOREIGN TABLE "type_UUID+" ALTER COLUMN "u" OPTIONS (SET column_type 'BLOB');
+--Testcase 169:
+DELETE FROM "type_UUID" WHERE "u" = 'a0eebc99-9c0b4ef8-bb6d6bb9-bd380a11';
+--Testcase 170:
+SELECT * FROM "type_UUID+";
+ i | u | t | l 
+---+---+---+---
+(0 rows)
+
+--Testcase 171:
+ALTER FOREIGN TABLE "type_UUID+" ALTER COLUMN "u" OPTIONS (SET column_type 'TEXT');
+--Testcase 172:
+DELETE FROM "type_UUID" WHERE "u" = 'a0eebc99-9c0b4ef8-bb6d6bb9-bd380a12';
+--Testcase 173:
+SELECT * FROM "type_UUID+";
+ i | u | t | l 
+---+---+---+---
+(0 rows)
+
+--Testcase 174:
+INSERT INTO "type_UUID" ("i", "u") VALUES (41, '{b0eebc99-9c0b4ef8-bb6d6bb9-bd380a15}');
+--Testcase 175:
+SELECT * FROM "type_UUID+" WHERE "i" = 41;
+ i  |                  u                   |  t   | l  
+----+--------------------------------------+------+----
+ 41 | b0eebc99-9c0b-4ef8-bb6d-6bb9bd380a15 | blob | 16
+(1 row)
+
+--Testcase 176:
+UPDATE "type_UUID" SET "u" = '{b0eebc99-9c0b4ef8-bb6d6bb9-00000a15}' WHERE "u" = '{b0eebc99-9c0b4ef8-bb6d6bb9-bd380a15}';
+--Testcase 177:
+SELECT * FROM "type_UUID+";
+ i  |                  u                   |  t   | l  
+----+--------------------------------------+------+----
+ 41 | b0eebc99-9c0b-4ef8-bb6d-6bb900000a15 | blob | 16
+(1 row)
+
+--Testcase 178:
+ALTER FOREIGN TABLE "type_UUID" ALTER COLUMN "u" TYPE bytea;
+--Testcase 179:
+INSERT INTO "type_UUID" ("i", "u") VALUES (42, decode('a0eebc999c0b4ef8bb6d6bb9bd380a11f1', 'hex'));
+--Testcase 180:
+INSERT INTO "type_UUID" ("i", "u") VALUES (43, decode('b0eebc999c0b4ef8bb6d6bb9bd380a', 'hex'));
+--Testcase 181:
+INSERT INTO "type_UUID" ("i", "u") VALUES (44, decode('b0eebc999c0b4ef8bb6d6bb9bd380a1', 'hex'));
+ERROR:  invalid hexadecimal data: odd number of digits
```

Comment:
Invalid hexa literal, please correct it. Or can you add comment for test purpose?

Reply:
This is real test case, but more about PostgreSQL behavior than about `sqlite_fdw`.

Reply:
Please remove this test, we do not need check PostgreSQL behavior here.  It has no meaning for sqlite_fdw

Reply:
Removed, please verify.

Reply:
Thank you. Confirmed.

Reply:
Resolved.